docs/src/infrastructure/architecture_decision_records/deploying_LiteLLM.md

```diff
@@ -0,0 +1,85 @@
+Architecture Decision Record: Adding LiteLLM to Matrix Infrastructure Kubernetes Stack
+Date: 04-09-2025
+Status: Proposed
+Author: Platform Engineering Team
+
+# Context
+
+The MATRIX platform at EveryCure requires a robust LLM gateway to manage multiple AI model providers efficiently. As we expand our AI-driven drug repurposing capabilities using Gemini 2.0 and other LLM providers, we need a unified interface that provides:
```

Comment:
generally the ADR discusses how to get litellm up and running but does not describe anything about _how we should use it_

- which models do we expose through here
- do 

E.g. we can expose a vertex-ai based anthropic model through litellm

<img width="1449" height="941" alt="Image" src="https://github.com/user-attachments/assets/03b5609c-1032-4b74-863e-32b964d784ce" />